package.json

```diff
@@ -60,10 +61,12 @@
                     "to": ".",
                     "filter": "*.so*"
                 }
-            ]
+            ],
+            "category": "Development"
         },
         "mac": {
             "category": "public.app-category.developer-tools",
+            "artifactName": "${name}-${version}-${os}.${ext}",
```

Comment:
It seems the different artifacts use different naming schemes. Could they be more inline with each other and with our versioning scheme?

Reply:
This makes it backwards compatible with our existing artifacts.